src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -151,11 +151,12 @@ class WorkspaceReimburseView extends React.Component {
             rateValue: numValue.toFixed(3),
         });
 
-        Policy.setCustomUnitRate(this.props.policyID, this.state.unitID, {
-            customUnitRateID: this.state.rateID,
-            name: this.state.rateName,
-            rate: numValue.toFixed(3) * 100,
-        }, null);
+        const distanceCustomUnit = _.find(lodashGet(this.props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        const currentCustomUnitRate = lodashGet(distanceCustomUnit, ['rates', this.state.unitRateID], {});
+        Policy.setCustomUnitRate(this.props.policyID, currentCustomUnitRate, this.state.unitID, {
+            ..._.omit(currentCustomUnitRate, 'rate'),
+            rate: numValue * CONST.POLICY.CUSTOM_UNIT_RATE_BASE_OFFSET,
```

Comment:
The previous code called `.toFixed(3)` on the `numValue`. JS is dumb 😅, so without it we get this:
<img width="1482" alt="Screen Shot 2022-08-30 at 4 02 46 PM" src="https://user-images.githubusercontent.com/7277067/187459309-5709f468-adcb-4d71-8c19-7ecc7b61c55a.png">

So I think it should be added back, or abstracted earlier in the function (since it's called when updating the state value as well).